### ISSUE TITLE
About the latest release

### ISSUE THREAD

**Issue by chenrui333** [vmware/govmomi#3179]
👋 it looks like [0.30.6](https://github.com/vmware/govmomi/releases/tag/v0.30.6) did not complete successfully, raise this issue for some awareness. Thanks!

relates to https://github.com/Homebrew/homebrew-core/pull/136845

**Comment by lubronzhan** [comment]
Related error 
https://github.com/vmware/govmomi/actions/runs/5579622016/job/15109828705
```
  • checking distribution directory
  • loading go mod information
  ⨯ release failed after 0s                  error=failed to get module path: exit status 2: flag provided but not defined: -m
usage: list [-e] [-race] [-f format] [-json] [-tags 'tag list'] [packages]

List lists the packages named by the import paths, one per line.

The default output shows the package import path:
```

So the error comes from golang 
```
go list -m
```
From above, it doesn't have `-m` flag 

This is weird. because golang 1.19 and 1.20 both have this flag. And [here](https://github.com/vmware/govmomi/blob/main/.github/workflows/govmomi-release.yaml#L75) it set golang to 1.20.

**Comment by lubronzhan** [comment]
Hi @douglas-c I think it might be related to this change
https://github.com/vmware/govmomi/commit/64aafeb9531695d6fc1f01a48233772a10859bc9#diff-0dc59770f4c9ccde5a701feabfb11367538adc70009d651beacd0815ff150bf4R75
Maybe try use 1.19 [here](https://github.com/vmware/govmomi/commit/64aafeb9531695d6fc1f01a48233772a10859bc9#diff-0dc59770f4c9ccde5a701feabfb11367538adc70009d651beacd0815ff150bf4R75) ?

**Comment by lubronzhan** [comment]
Ok found it here. This is golang 1.2.2?????

```

5s
Run actions/setup-go@v3
Setup go version spec [1](https://github.com/vmware/govmomi/actions/runs/5579622016/job/15109828705#step:5:1).2
Attempting to download 1.2...
matching 1.2...
Not found in manifest.  Falling back to download directly from Go
Install from dist
Acquiring go1.2.2 from https://storage.googleapis.com/golang/go1.2.2.linux-amd64.tar.gz
Extracting Go...
/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/d074[9](https://github.com/vmware/govmomi/actions/runs/5579622016/job/15109828705#step:5:10)a70-3b15-47a5-a191-9ec8b14e0b44 -f /home/runner/work/_temp/2b30b36e-8e14-4025-a181-4c50617c83[10](https://github.com/vmware/govmomi/actions/runs/5579622016/job/15109828705#step:5:11)
Successfully extracted go to /home/runner/work/_temp/d0749a70-3b15-47a5-a191-9ec8b[14](https://github.com/vmware/govmomi/actions/runs/5579622016/job/15109828705#step:5:15)e0b44
Adding to the cache ...
Successfully cached go to /opt/hostedtoolcache/go/1.2.2/x64
Added go to the path
Setting GOROOT for Go version < 1.9
Successfully set up Go version 1.2
go version go1.2.2 linux/amd64

go env
```

**Comment by lubronzhan** [comment]
Oh need to quote https://github.com/actions/setup-go/issues/328

**Comment by lubronzhan** [comment]
Hi @dougm just a reminder, after the fix, I think you still need to cut 0.30.6?
